### DISCUSSION TITLE
Seeking Community Feedback on Drafted NetworkX Organization Landing Page

### DISCUSSION THREAD

**Discussion by axif0** [networkx/networkx#7022] [General]
Hello NetworkX Community,

I have drafted a README for the NetworkX organization's landing page and would love to hear your thoughts and feedback. The draft README can be found here: [Draft README](https://github.com/networkx/.github/pull/1)

The README includes the following sections:
- An introduction to NetworkX
- Links to key resources such as the website, source code, bug reports, tutorial, GitHub Discussions, and mailing list
- Detailed information about contributing to NetworkX
- Code of Conduct

The aim of this README is to provide a welcoming introduction to NetworkX for new visitors and to make important resources readily accessible for everyone.

Please share your thoughts on the following:
- What do you expect to see when you visit the NetworkX organization's landing page?
- Is there any additional information or resources that you think should be included in the README?
- Do you have any suggestions for improving the current draft?

Your feedback is greatly appreciated and will help make the landing page more useful and informative for the entire community.

Thank you in advance for your time and input!

**Comment by jarrodmillman** [comment]
I am -1 on this. It will take up space on the top of the org page, which will make the org page less usable for me.

**Comment by dschult** [comment]
I am also -1 on this.  
The landing page for the project is networkx.org
The github landing page for networkx is the main repo: networkx/networkx
And two pinned repos in the github org page are the ones we want people to know about: nx-guides and networkx
But I'd rather not duplicate information that would inevitably become out of date.

**Comment by axif0** [comment]
Thank you for your kind feedback @jarrodmillman and @dschult.

I understand networkx is the main repo for networkx organization,  however, as a landing page serves as the organization's face, an engaging and informative introduction to the organization's purpose, projects, and community would help newcomers to navigate what is where...rather than having these information in networkx's readme and overwhelming that file. 
What do you think about it?

**Reply by jarrodmillman** [reply]
I am -1 on this.

**Reply by jarrodmillman** [reply]
I also don't want to add to the README.md file. I believe it already contains everything that it should. If you are interested in contributing to NetworkX, I would recommend working on bug-fixes or new features, rather than focusing on the project's landing page or README.md.

**Comment by dschult** [comment]
I don't think the organization's github page is a landing page for us. 
The `networkx/networkx` github page is a landing page for source code. 
The primary landing page for the package is `networkx.org`.

**Comment by axif0** [comment]
I appreciate your feedback on the proposed changes to the NetworkX organization landing page. Your perspectives have given me a lot to consider, and it's clear from your responses and the community's votes that my proposal does not align with the current needs and direction of the project.

Closing this discussion now.
